### PR TITLE
now we are correctly rendering the list of members for an organization

### PR DESCRIPTION
**Pull request by jac1013**
## Story / Task

[One-98](https://bixlabs.atlassian.net/browse/ONE-98)

## What this PR does?

It adds all the necessary things to show the list of members of an organization.

### How to test / reproduce
* Go to: ``school/0beeebbc79c11000/members``

### TODOs
- We need to fix the template and the component to show the information of the user after the backend guys provide us with an improve response from the server.
- We need to attach this route to the correct button where this functionality will be triggered, because we don't have the new design yet this is not attach to anything for now.

Thanks!

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -322,16 +322,16 @@ module.exports = {
...
+    // new UglifyJsPlugin({
+    //   "mangle": {
+    //     "screw_ie8": true
+    //   },
+    //   "compress": {
+    //     "screw_ie8": true,
+    //     "warnings": false
+    //   },
+    //   "sourceMap": false
+    // }),
```
Comment (wuilliam321) [inline webpack.config.js]: If we does not need this we can remove, what do you think?

File: src/app/school/member/member-list-item/organization-members-list-item.component.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    <div class="member__name-and-role-container">
+        <div>
+            <b class="member__name-and-role__name">{{member.userId}}</b>
+            <button class="btn btn-default member__options-btn">
+                <span aria-hidden="true" class="glyphicon glyphicon-option-vertical"></span>
+            </button>
+            <button class="member__follow-member">Follow</button>
+        </div>
+        <div class="member__name-and-role__role-name-container">
+            <small class="member__name-and-role__role-name">{{showRoles(member.roles)}}</small>
```
Comment (wuilliam321) [inline src/app/school/member/member-list-item/organization-members-list-item.component.html]: Excellent, but, it is recommended to prepare data on component and use something like `{{member.role}}`. Leave as is! it is excellent! very readable.

**Comment by wuilliam321** [comment]
Rebase and i think it is ready

File: src/app/school/member/shared/organizationMembers.service.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        const options = new RequestOptions({ withCredentials: true });
+
+        return <Observable<OrganizationMembersResponse>>this.http.get(URL, options)
+            .map((response: Response) =>  response.json())
+            .catch(this.exceptionService.catchBadResponse);
+    }
+}
+
+@Injectable()
+export class OrganizationMembersResolver implements Resolve<Observable<OrganizationMembersResponse>> {
```
Comment (evasquez26) [inline src/app/school/member/shared/organizationMembers.service.ts]: I don't know if we should have this resolver here or in a separate file, what do you think?

File: webpack.config.js (hunk 1 above)
Comment (evasquez26) [inline webpack.config.js]: Does removing this helps to find erros? Seems to keep uglifying to me idk

File: src/app/app.routing.module.ts [hunk 4]
```diff
@@ -76,6 +78,13 @@ const routes: Routes = [
     component: SchoolCreateComponent
   },
   {
+    path: 'school/:organizationId/members',
```
Comment (evasquez26) [inline src/app/app.routing.module.ts]: I think this is not going to be routable anymore when moving it to the tab of the schools

File: src/app/school/school-details/school-details.component.html [hunk 5]
```diff
@@ -179,6 +179,7 @@ <h2 class="course__title"><a [routerLink]="['/course-details/', course.id]">{{ c
       <!--/COURSES -->
       <!-- MEMBERS-->
       <div class="tab-pane" [ngClass]="{'active':activeTab==='members'}">
+        <organization-members-list></organization-members-list>
```
Comment (evasquez26) [inline src/app/school/school-details/school-details.component.html]: I think that with this implementation it's hard to tell what the component receives unless we check the `school-details.component`, also this component calls another one, I think just one component receiving members as input will be less complicated, what do you think?

File: src/app/app.routing.module.ts [hunk 6]
```diff
@@ -248,5 +252,6 @@ export const routableComponents = [
   TermsComponent,
   CourseComponent,
   CourseDetailsComponent,
-  CourseCategoriesComponent
+  CourseCategoriesComponent,
+  OrganizationMembersComponent
```
Comment (evasquez26) [inline src/app/app.routing.module.ts]: As this is not a routable component anymore, it should go to app.module (for now)

File: webpack.config.js (hunk 1 above)
Comment (jac1013) [inline webpack.config.js]: It should not uglify after this @evasquez26

File: src/app/school/member/member-list-item/organization-members-list-item.component.html (hunk 2 above)
Comment (jac1013) [inline src/app/school/member/member-list-item/organization-members-list-item.component.html]: I'll do this, thnaks!

**Pull request merged**